### ISSUE TITLE
World crashes on loading, caused by unknown mod

### ISSUE THREAD

**Issue by T4UC3T1** [GTNewHorizons/GT-New-Horizons-Modpack#11820]
### Your Pack Version

2.2.0.0

### Your Server

SP

### Type of Server

Single Player

### Your Actions

When loading onto my save, the game crashes right after "Building Terrain" and generates this crash report. It identifies no mod as the cause for the crash. Backups of the world refuse to load as well.

### Crash Report

[crash-2022-11-13_01.36.38-client.txt](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/files/9995543/crash-2022-11-13_01.36.38-client.txt)

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the crash still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this crash consistently by following the exact steps I described above.
- [ ] I have asked other people and they confirm they also crash by following the exact steps I described above.

**Comment by chochem** [comment]
no, thats not really any mod. Afaik thats a vanilla bug that nobody ever really figured out. Just load again, it should be random.

**Comment by T4UC3T1** [comment]
I wasnt using RemoteIO anywhere as i had barely got into the steam age.
